views/cardsList.pug

```diff
@@ -171,6 +171,19 @@ block content
         div.text-center
           <div id="ezoic-pub-ad-placeholder-108"></div>
 
+      -
+        function getName(d) {
+          var lang = t("lang");
+          if (lang === "en") {
+            return d.name;
+          } else if (lang === "ja") {
+            return d.ja_name;
+          } else if (lang === "zh") {
+            // return d.zh_name;
+            return d.name;
+          }
```

Comment:
Function seems similar to another piece of code. Is it possible to move the common code somewhere and access it from both places?